Servers/swagger.yaml

```
@@ -2355,4 +3183,4 @@ components:
         review_date:
           type: string
           format: date-time
-          example: "2024-10-20T00:00:00.000Z"
+          example: "2024-10-20T00:00:00.000Z"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)